aptos-move/framework/aptos-framework/sources/account/permissioned_delegation.move

```diff
@@ -201,7 +201,7 @@ module aptos_framework::permissioned_delegation {
             signature,
         };
         let auth_data = auth_data::create_auth_data(vector[1, 2, 3], bcs::to_bytes(&sig_bundle));
-        assert!(!is_permissioned_signer(&account), 1);
+        assert!(!is_permissioned_signer(&account));
```

Comment:
The error codes can be useful, but that's fine I guess

Reply:
With stack trace, these hardcoded error codes should no longer be necessary

Reply:
Then we should probably get rid of these as a whole later.